src/forms.ts

```diff
@@ -14,7 +17,10 @@ enum FormErrorCodeEnum {
   FORM_NOT_FOUND = 'FORM_NOT_FOUND',
   NO_FILE_UPLOADS = 'NO_FILE_UPLOADS',
   TOO_MANY_FILES = 'TOO_MANY_FILES',
-  FILES_TOO_BIG = 'FILES_TOO_BIG'
+  FILES_TOO_BIG = 'FILES_TOO_BIG',
+  STRIPE_SCA_ERROR = 'STRIPE_SCA_ERROR',
+  STRIPE_PROMISE_ERROR = 'STRIPE_PROMISE_ERROR',
+  PAYMENT_METHOD_ERROR = 'PAYMENT_METHOD_ERROR'
```

Comment:
What if we used a "STRIPE_CLIENT_ERROR" instead and used it for both payment method errors and stripe promise errors?

Reply:
True, will work on it!